modules/cluster-logging-eventrouter-deploy.adoc

```diff
@@ -11,7 +11,7 @@ The following Template object creates the Service Account, ClusterRole, and Clus
 
 [NOTE]
 ====
-To use the Event Router, you must enable JSON log parsing using the `MERGE_JSON_LOG` feature. For important information `MERGE_JSON_LOG`, see
+Deploying the Event Router automatically enables JSON log parsing using the `MERGE_JSON_LOG` feature. For important information `MERGE_JSON_LOG`, see
 xref:../logging/config/cluster-logging-fluentd.html#cluster-logging-fluentd-json_cluster-logging-fluentd[Configuring log collection JSON parsing].
 You must set cluster logging to the unmanaged state to use `MERGE_JSON_LOG`.
```

Comment:
Instead of this, say something like
```
You must set `TRANSFORM_EVENTS=true` in order to process and store eventrouter events in Elasticsearch.  In order to do this, you must set cluster logging to the unmanaged state.
```
The add a link to the docs where we explain how to set cluster logging to the unmanaged state.  Then
```
oc set env ds/fluentd TRANSFORM_EVENTS=true
```